norfair/camera_motion.py

```diff
@@ -151,7 +153,11 @@ def abs_to_rel(self, points: np.ndarray):
         points_transformed = points_transformed / points_transformed[:, -1].reshape(
             -1, 1
         )
-        return points_transformed[:, :2]
+        new_points_transformed = points_transformed[:, :2]
+        if np.isnan(new_points_transformed).any() or np.isinf(new_points_transformed).any():
+            new_points_transformed = np.array([[0, 0], [0, 0]])
+
+        return new_points_transformed
 
     def rel_to_abs(self, points: np.ndarray):
```

Comment:
I guess the same error that happened in `abs_to_rel` could also happen in `rel_to_abs` because of doing essentially the same computation but with the inverse matrix, so the fix should also be applied here. Is there any reason for not fixing it here that I am missing?